docs/src/modules/components/AppFrame.js

```diff
@@ -141,6 +142,16 @@ const styles = (theme) => ({
       display: 'none',
     },
   },
+  banner: {
+    display: 'block',
+    padding: 4,
+    textAlign: 'center',
+    backgroundColor: theme.palette.primary.dark,
```

Comment:
Don't feel like this one is working too well on dark mode... I've tried using `palette.background.paper` and it worked better.

Reply:
Should we use the current for light mode and the grey for dark?

Reply:
I think it seems to work fine?!